Miniforge3/construct.yaml

```diff
@@ -19,12 +19,17 @@ keep_pkgs: True
 license_file: ../LICENSE
```

Comment:
Can we somehow pin the version of `mamba` too here?

Over in my PR I used the following for that
```
{% if name.startswith("Mamba") %}
{% set version = os.environ.get("MINIFORGE_VERSION", "4.9.2-0.7.3-0") %}
{% else %}
{% set version = os.environ.get("MINIFORGE_VERSION", "4.9.2-0") %}
{% endif %}
```

Reply:
No, that's not how `MINIFORGE_VERSION` is set. It's deduced from the git tag.